feat(content): Mining Update - Assets + Balance (#9621)
endless-sky/endless-sky: data/human/weapons.txt

	"random velocity" 1
	"velocity scale" -.007

outfit "Mining Laser"
	category "Guns"
	cost 38000
	thumbnail "outfit/mining laser"
	"mass" 12
	"outfit space" -12
	"weapon capacity" -12
	"gun ports" -1
	weapon
		sprite "projectile/mining laser"
			"frame rate" 1
		sound "mining laser"
		"hit effect" "beam laser impact"
		"inaccuracy" .35
		"velocity" 200
		"lifetime" 1
		"reload" 1
		"firing energy" 1.1
		"firing heat" 2.1
		"shield damage" .8
		"hull damage" 1.4
		"minable damage" 1.9
		"prospecting" 1

> 	description "The development of the Beam Laser in the mid-2400s not only led to a revolution in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves. The modern Mining Laser is a derivative of the Beam Laser engineered to optimise mineral yield and extraction rates at the expense of range and shield damage."

Reviewer: Let's break the lore across both mining weapons. This allows us to keep the lengths down as well.
```suggestion
	description "The development of the Beam Laser in the mid-2400s not only led to a revolution in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves."
```

Author: Oops

> 	description "The development of the Beam Laser in the mid-2400s not only led to a revolution in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves. The modern Mining Laser is a derivative of the Beam Laser engineered to optimise mineral yield and extraction rates at the expense of range and shield damage."

Reviewer 2: ```suggestion
	description "The development of the Beam Laser in the mid-2400s not only led to a revolution in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves. The modern Mining Laser is a derivative of the Beam Laser engineered to optimize mineral yield and extraction rates at the expense of range and shield damage."
```
Oops.

> 	description "The development of the Beam Laser in the mid-2400s not only led to a revolution in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves. The modern Mining Laser is a derivative of the Beam Laser engineered to optimize mineral yield and extraction rates at the expense of range and shield damage."

Reviewer: Let's break the lore across both mining weapons. This allows us to keep the lengths down as well.
```suggestion
	description "The development of the Beam Laser in the mid-2400s not only led to a revolution in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves."
```

Reviewer 2: I think we should have something alluding to the Mining Laser's prospecting attribute, which this would remove.

Reviewer: Yeah. If we're to mention it in either, it probably does make sense for it to be in the gun's description since it's more common and located wherever the turret is found.

Reviewer: ```suggestion
	description "The development of the Beam Laser in the mid-2400s not only led to advancements in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves. The modern Mining Laser is a derivative of the Beam Laser engineered to optimize mineral yield and extraction rates at the expense of range and shield damage."
```

Reviewer 2: ```suggestion
	description "The development of the Beam Laser in the mid-2400s not only led to advancements in space warfare, but also in the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves."
	description "	The modern Mining Laser is a derivative of the Beam Laser engineered to optimize mineral yield and extraction rates at the expense of range and shield damage."
```
I think "advancements in the field of asteroid mining" makes more sense than "advancements to the field of asteroid mining". At the very least it will be consistent.

> 	description "	The Mining Laser also comes equipped with an extremely short range asteroid targeting scanner. As such, drones and fighters that have it installed will automatically engage in mining when launched."

Reviewer: There is no such attribute on this weapon.
```suggestion
```

outfit "Mining Laser Turret"
	category "Turrets"
	cost 66000
	thumbnail "outfit/mining laser turret"
	"mass" 18
	"outfit space" -18
	"weapon capacity" -18
	"turret mounts" -1
	"required crew" 1
	weapon
		sprite "projectile/mining laser"
			"frame rate" 1
		"hardpoint sprite" "hardpoint/mining laser turret"
		"hardpoint offset" 0.7 8.
		sound "mining laser"
		"hit effect" "beam laser impact"
		"inaccuracy" .35
		"turret turn" 3.8
		"velocity" 200
		"lifetime" 1
		"reload" 1
		"firing energy" 1.1
		"firing heat" 2.1
		"shield damage" .8
		"hull damage" 1.4
		"minable damage" 1.9
		"prospecting" 1

> 	description "The efficiency of a Mining Laser is truly brought out when mounted on a turreted base, which makes it easier for large and slow ships with spacious cargo holds to mine."
> 	description "	Mining Laser Turrets are designed to be used in conjunction with Asteroid Scanners, as being unable to target asteroids means you have to fire the weapon manually as if it were a fixed gun."

Reviewer: Aim manually, not fire manually.
```suggestion
	description "The modern Mining Laser is a derivative of the Beam Laser engineered to optimize mineral yield and extraction rates at the expense of range and shield damage. While asteroid miners typically prefer more nimble ships, the Mining Laser Turret makes it easier for large and slow ships with spacious cargo holds to mine."
	description "	Mining Laser Turrets are designed to be used in conjunction with asteroid scanners, as being unable to target asteroids means you have to aim the weapon manually as if it were a fixed gun."
```

outfit "Tractor Beam"
	category "Turrets"
	cost 75000
	thumbnail "outfit/tractor beam"
	"mass" 22
	"outfit space" -22
	"weapon capacity" -22
	"turret mounts" -1
	weapon
		"hardpoint sprite" "hardpoint/tractor beam"
		"hardpoint offset" 4.
		sound "tractor beam"
		"hit effect" "tractor beam"
		"tractor beam" 7
		"velocity" 300
		"lifetime" 1
		"reload" 1
		"firing energy" 2.1
		"firing heat" .8

> 	description "The anti-gravity properties of repulsor engines that allow ships to escape the gravity well of planets, without the difficulty that humanity once faced before the era of the hyperdrive, started a revolution in human spaceflight during the 24th century. It was not long before this technology was put to use elsewhere."
> 	description "	Tractor beams utilize repulsor technology to allow ships to attract space debris to them, be that the ores from a recently mined asteroid, or the dumped cargo from a merchant ship fleeing for its life. This has led to tractor beams being a popular outfit among asteroid miners and pirates alike."

Reviewer: Shortening up this one too.
```suggestion
	description "The anti-gravity properties of repulsor engines started a revolution in human spaceflight during the 24th century, and it was not long before this technology was put to use elsewhere. Tractor beams utilize repulsor technology to allow ships to attract space debris to them, be that the ores from a recently mined asteroid, or the dumped cargo from a merchant ship fleeing for its life. This has led to tractor beams being a popular outfit among asteroid miners and pirates alike."
```

Reviewer 2: Haven't read it all but I'm noticing a lot of "revolutions" in outfit descriptions (partly my fault...) We should find a different phrase for some variation.

Reviewer: Changed my suggestion to the mining laser gun to swap "revolution" for "advancement."

	"random velocity" 1
	"velocity scale" -.007

outfit "Mining Laser"
	category "Guns"
	cost 38000
	thumbnail "outfit/mining laser"
	"mass" 12
	"outfit space" -12
	"weapon capacity" -12
	"gun ports" -1
	weapon
		sprite "projectile/mining laser"
			"frame rate" 1
		sound "mining laser"
		"hit effect" "beam laser impact"
		"inaccuracy" .35
		"velocity" 200
		"lifetime" 1
		"reload" 1
		"firing energy" 1.1
		"firing heat" 2.1
		"shield damage" .8
		"hull damage" 1.4
		"minable damage" 1.9

> 		"prospecting" 0.1

Reviewer 3: ```suggestion
		"prospecting" 1
```

	description "	This mining laser comes equipped with an extremely short range asteroid targeting scanner as well as mining algorithms. As such, drones and fighters that have it installed will automatically engage in mining when launched."

outfit "Mining Laser Turret"
	category "Turrets"
	cost 66000
	thumbnail "outfit/mining laser turret"
	"mass" 18
	"outfit space" -18
	"weapon capacity" -18
	"turret mounts" -1
	"required crew" 1
	weapon
		sprite "projectile/mining laser"
			"frame rate" 1
		"hardpoint sprite" "hardpoint/mining laser turret"
		"hardpoint offset" 0.7 8.
		sound "mining laser"
		"hit effect" "beam laser impact"
		"inaccuracy" .35
		"turret turn" 3.8
		"velocity" 200
		"lifetime" 1
		"reload" 1
		"firing energy" 1.1
		"firing heat" 2.1
		"shield damage" .8
		"hull damage" 1.4
		"minable damage" 1.9

> 		"prospecting" 0.1

Reviewer 3: ```suggestion
		"prospecting" 1
```
